hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/lifecycle/OMLifecycleConfigurationSetRequest.java

```diff
@@ -100,7 +99,7 @@ public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
             .setVolume(resolvedBucket.realVolume())
             .setBucket(resolvedBucket.realBucket());
 
-    newLifecycleConfiguration.setCreationTime(Time.now());
+    newLifecycleConfiguration.setCreationTime(System.currentTimeMillis());
```

Comment:
Nit: This is equivalent? In that case, there is no need to change I think.

Reply:
Equivalent, I just unified the writing here